### ISSUE TITLE
It should be clear where to file an issue

### ISSUE THREAD

**Issue by jasonLaster** [css-modules/css-modules#27]
The project is split between many different repos. Yet, at this stage many of the issues are issue specific so it's not clear where they should be filed. 

I think the READMEs should point to one place to file the issues. The best solution, would be to have a clear spec or list of supported features. That way, that repo could have issues and PRs for updating the spec. 

Ideally, an update to the spec could list the degree of support. So for example, a new un-implemented feature would be mentioned in the spec as `in-progress`. And a completed feature, would be mentioned in the spec as `implemented`.

**Comment by geelen** [comment]
Yeah, that could be clearer. FYI, this is the project for discussions on the standard. I'm not too interested in listing the features that are in/out though. The goal is to have as few features as possible but solve the most number of problems. CSS Modules is 100% postcss transforms compiling to ICSS, so hopefully that allows folks to customise their own builds to suit their needs without needing the whole CSS Modules spec to change.

FYI, we do have one spec change PR open around media queries (though I've broken my own rule and it's against the loader-core project!): https://github.com/css-modules/css-modules-loader-core/pull/18
